DatadogInternal/Sources/NetworkInstrumentation/SpanID.swift

```diff
@@ -0,0 +1,161 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2019-2020 Datadog, Inc.
+ */
+
+import Foundation
+
+public struct SpanID: RawRepresentable, Equatable, Hashable {
+    public static let invalidId: UInt64 = 0
```

Comment:
I was wondering when reading tests how it's been done 🤔 

Great idea! 👍 - improves readability!